rooms.txt

```diff
@@ -1,7 +1,11 @@
 Hallway
-Español
-Français
-Deutsche
-中文
-हिन्दी
-TheBackrooms
\ No newline at end of file
+Grand Hall Room
```

Comment:
Can you explain your reasoning in the changes to the default rooms list?

Reply:
Yeah, I just wanted something standard and organized and people can have rooms for them, they select which one, and go into it. Keeping it in a similar convention would be nice imo.

Reply:
I'm not sure having room names like these would be very useful, they seem like they'd be mostly empty - I see people who are english-speaking gravitate to one main public room in other online chats.

There's the idea I had of using rooms with titles of different languages to avoid having people speaking over each other to talk in different languages - people who are chinese will go to the chinese room. It's there because it's a problem I've seen in some online chats

Reply:
Additionally:
- Calling a room `English chat` seems redundant, you're already at a chat website. Why isn't `Non-English` called `Non-English chat` for consistency?
- `Private-1` would be the name for a room that is **publicly** advertised on the frontpage, calling it private would be misleading because other strangers would be able to join
- I don't think `Others` is going to be a very popular room people will want to use
- "TheBackrooms" is a reference to a weird looking photo from some online meme, wanted to leave that in as a fun little room, it's good a secondary public English-speaking room

Reply:
Ahan, I'll change the names then :D